src/metabase/automagic_dashboards/core.clj

```diff
@@ -1070,41 +1069,37 @@
 
 (defn- pluralize
   [x]
-  (case (mod x 10)
+  (case (int (mod x 10))
```

Comment:
Why int cast? Does `u.date/extract`  return a float for `:week-of-year`?

Reply:
If you enable reflection warnings it complains that you're using `case` with integer conditions but it doesn't know if the value it's comparing is an integer or something like that. So this is mostly to get it to stop complaining when I turn `*warn-on-reflection*` on in the REPL. I'll add a comment

Reply:
This is the error 

```
Performance warning, /Users/cam/metabase/src/metabase/automagic_dashboards/core.clj:1071:3 - case has int tests, but tested expression is not primitive.
```